index.html

```diff
@@ -4,34 +4,65 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="stylesheet" href="index.css">
+    <title>Project 1</title>
+    <style>
+        h1,
+        h3 {
+            font-family: Arial, Helvetica, sans-serif;
+            color: #f4e1d2;
+        }
+
+        h1 {
+            text-decoration: underline;
+        }
+
+        h3 {
+            font-style: italic;
+        }
+
+        p.hi {
+            font-size: 15px;
+        }
+    </style>
 </head>
 
 <body>
-    on paper
-    by Jacqueline Woodson
+    <div class="container" id="container">
+        <div id="container3">
+            <h1 id="text2">On Paper</h1>
+            <h3 id="text3">by Jacqueline Woodson</h3>
+            <div id="container2">
+                <pre style="word-wrap:break-word; word-wrap:white-space; word-wrap:pre-wrap;color:#f4e1d2" id="hi"
+                    class="text1">
+            on paper
+            by Jacqueline Woodson
 
-    The first time I write my full name
+            The first time I write my full name
 
-    Jacqueline Amanda Woodson
+            Jacqueline Amanda Woodson
 
-    without anybody's help
-    on a clean white page in my composition notebook,
-    I know
+            without anybody's help
+            on a clean white page in my composition notebook,
+            I know
 
-    if I wanted to
+            if I wanted to
 
-    I could write anything.
+            I could write anything.
 
-    Letters becoming words, words gathering meaning,
-    becoming
-    thoughts outside my head
+            Letters becoming words, words gathering meaning,
+            becoming
+            thoughts outside my head
 
-    becoming sentences
+            becoming sentences
 
-    written by
+            written by
 
-    Jacqueline Amanda Woodson
+            Jacqueline Amanda Woodson
+        </pre>
+            </div>
+        </div>
+    </div>
 </body>
```

Comment:
It looks like you likely have some extraneous HTML in your project; see [this video](https://www.loom.com/share/3b5b64169ac847c3aa956f41dfa49105) for how to simplify your HTML 👍🏻